how-to/common/package.json

```diff
@@ -25,20 +25,20 @@
 	],
 	"license": "SEE LICENSE IN LICENSE.MD",
 	"dependencies": {
-		"@openfin/core": "^28.72.17",
+		"@openfin/core": "^29.73.14",

```

Comment:
The main branch is still workspace 10 which is bound to runtime 28.72.17. The v11 branch should have this 29 version but not main/v10 branches (otherwise there may be a feature that the types indicate is there but the runtime doesn't support).